docs/dashboards/README.md

```diff
@@ -0,0 +1,163 @@
+
+This folder contains dashboards that can be used to monitor the health of deployed OpenTelemetry collectors. The dashboards are in json format and can be uploaded to your Dynatrace tenant by [following the steps in the Dynatrace documentation](https://docs.dynatrace.com/docs/observe-and-explore/dashboards-and-notebooks/dashboards-new/get-started/dashboards-manage#dashboards-upload).
+
+For Kubernetes, two dashboards exist: 
+- [collector_selfmon_kubernetes_all.json](collector_selfmon_kubernetes_all.json): Shows aggregated data for all collectors sending data.
+- [collector_selfmon_kubernetes_single.json](collector_selfmon_kubernetes_single.json): Allows to drill down into a single collector based on the collectors service name and pod name.
+
+The dashboards rely on metrics from the collectors' [internal telelemetry](https://opentelemetry.io/docs/collector/internal-telemetry/). See the [list of internal metrics](https://opentelemetry.io/docs/collector/internal-telemetry/#lists-of-internal-metrics) for an overview of which metrics are available.
+
+## Prerequisites
+The dashboards rely on the selfmonitoring capabilities of the OTel collector as well as certain attributes on the exported metrics data.
+Required attributes are: 
+- `service.name` (automatically added by the Collector)
+- `service.instance.id` (automatically added by the collector, needs to be added to the Dynatrace attribute allow list, see "[Adding `service.instance.id` to the allow list](#adding-serviceinstanceid-to-the-allow-list)")
+- `k8s.pod.name` (needs to be added to the telemetry data, see the [Kubernetes section](#kubernetes) below)
+
+### Adding `service.instance.id` to the allow list
+`service.name` and `k8s.pod.name` are on the Dynatrace OTLP allow list by default, `service.instance.id` is not. In order to add it follow [this guide](https://docs.dynatrace.com/docs/shortlink/metrics-configuration#allow-list) and add `service.instance.id` to the list.
+
+## Architecture
+Every OpenTelemetry collector has selfmonitoring capabilities, but they need to be activated.
+Self-monitoring data can be exported from the collector via the OTLP protocol.
+The suggested way of exporting selfmonitoring data is to run one collector dedicated for collecting and exporting the selfmonitoring data for the other running collectors, and forwarding that data to Dynatrace.
+Below, you can see a configuration example for a selfmonitoring collector.
+
+```yaml
+# receive selfmonitoring data via gRPC from OTel collector instances.
+receivers:
+  otlp/selfmon:
+    protocols:
+      grpc: 
+        endpoint: 0.0.0.0:4317
+
+processors:
+  # transform cumulative values to deltas. 
+  cumulativetodelta/selfmon: {}
+  # (kubernetes only) retrieves kubernetes attributes for other collectors sending to this collector. See Kubernetes prerequisites below.
+  k8sattributes/selfmon: {}
+
+  # prepend 'sfm.otelcol' to all selfmon metrics - the charts in the {dashboard_name}.json file expects this prefix.
```

Comment:
Might be worth pointing that out in the introduction above already for those who don't follow this snippet but already have a deployed collector and want to visualize its existing selfmon data.